packages/accordion/stories.tsx

```diff
@@ -95,7 +95,7 @@ storiesOf('Accordion Container', module)
           getPanelProps,
           expandedSections,
           disabledSections
-        }) => (
+        }: any) => (
```

Comment:
I'm going to type this argument value with the `IUseAccordionReturnValue` interface.